packages/rn-tester-e2e/README.md

```diff
@@ -0,0 +1,129 @@
+# RNTester E2E folder
+
+In this folder we have a local setup for doing E2E testing over RNTester via the usage of [Appium](https://appium.io/) and [WebDriverIO](https://webdriver.io/) and [Jest](https://jestjs.io/).
+
+- [Setting up](#setting-up)
+  - [(one-off) Setting up Appium](#one-off-setting-up-appium)
+  - [Building RNTester app](#building-rntester-app)
+    - [Building for iOS](#building-for-ios)
+    - [Building for Android](#building-for-android)
+  - [Setting up the RNTester E2E folder](#setting-up-the-rntester-e2e-folder)
+- [Testing the RNTester app E2E](#testing-the-rntester-app-e2e)
+- [Adding new tests (and project structure)](#adding-new-tests-and-project-structure)
+
+## Setting up
+
+### (one-off) Setting up Appium
+
+The first step you need to do is to ensure to install the tooling:
+
+```bash
+npm install appium@2.0.0-beta.71 -g
+appium driver install uiautomator2
+appium driver install xcuitest
+```
+
+> More details about drivers in Appium [here](https://appium.github.io/appium/docs/en/2.0/guides/managing-exts/) and [here](https://appium.github.io/appium/docs/en/2.0/quickstart/uiauto2-driver/)
+
+You should not need to run install commands for drivers separately more than once, even if you bump the dep in package.json.
+
+### Building RNTester app
+
+Building manually *.app* and *.apk* is required to run automation tests on local environment.
+
+0. *(optional)* If you previously built RNTester, you may need to clean up build files and Pods:
+
+    ```bash
+    yarn test-e2e-local-clean
+    ```
+
+1. Step 1: install node modules for the repository, then navigate in the rn-tester folder
+
+    ```bash
+    cd react-native
+    yarn install
+    cd packages/rn-tester
+    ```
+
+Now, depending on the platform, there are some specific steps
+
+#### Building for iOS
+
+0. Make sure you have Bundler `gem install bundler` - we use it ensure installing the right version of CocoaPods locally.
+1. Install Bundler and CocoaPods dependencies: `bundle install` then `bundle exec pod install` or `yarn setup-ios-hermes` for RNTester with Hermes. In order to use JSC instead of Hermes engine, run: `USE_HERMES=0 bundle exec pod install` or `setup-ios-jsc` instead.
+2. Open the generated `RNTesterPods.xcworkspace`.
+3. Build the app via XCode.
+4. Find the **RNTester.app** in `~/Library/Developer/Xcode/DerivedData/RNTesterPods-{id}/Build/Products/Debug-iphonesimulator`
```

Comment:
I think that we can explore how to tell `xcodebuild` to use a specific derivedData folder. At that point, we can automate these steps.

Reply:
There's `-derivedDataPath <path>` option for that (we're using this one internally in React Native CLI).

Reply:
yeah, I was referring to that. With that parameter, we can automate all these steps and move the build artifact in the right folder, simplifying the setup (and the CI, probably)

Reply:
Hi @kelset,
This is fixed in the last MR. Could you please resolve this comment?

Reply:
What do you mean: `This is fixed in the last MR`? I can still see that the instruction do not explain how to build from command line, passing the `--derivedDataPath`, so it does not seem fixed.

Reply:
Yeah, sorry for miscommunication. I added instructions to `README.md` how to build with RN CLI using `--buildFolder` flag (which corresponds to `-derivedDatpath`). Btw. I used this in CI, and it works great - saved a lot of time :D